tfidf_logreg.py

```diff
@@ -0,0 +1,38 @@
+# tfidf_logreg.py - Performs logistic regression on the phrase/sentiments with k-fold
+#                   cross-validation, and prints one-vs-one and one-vs-all
+#                   validation accuracy.
+#
+# Usage: python -B tfidf_logreg.py
+#
+# Wesley Chavez 08-27-18
+
+import config
+import pandas as pd
+import numpy as np
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.linear_model import LogisticRegression
+from sklearn.multiclass import OneVsRestClassifier
+from sklearn.multiclass import OneVsOneClassifier
+from sklearn.model_selection import cross_val_score
+
+def main():
+    train = pd.read_csv('~/Downloads/train.tsv', sep="\t")
```

Comment:
this won't work if I tried to run on my machine, for two reasons: 
1. since I don't have the dataset:
uploading the dataset of course problematic because it's a large file (and can sometimes be binary). I recommend git-lfs as a solution, thought there are other approaches you should research and consider.

2. I might have it in a different path. 
Try to avoid using '~' explicitly (`os.path.expanduser('~')` works in a pinch and is portable to windows)
`os.path.join` will join paths in an os independent way

Reply:
Good pointers.  The training/testing files are only 8.5 and 3.4 MB, do you think I should just add them to the project? Also, once I make these changes, what do we do with this pull request?

Reply:
I'm torn. They're small enough and text based that you can just add them to the repo, and git will happily track them. But this could be good opportunity to learn git-lfs and deal with this problem. 

Note that if they were binary files, then you'd have to use git-lfs or something that doesn't add them to git, because git doesn't know how to track binary files (https://development.robinwinslow.uk/2013/06/11/dont-ever-commit-binary-files-to-git/)

As you make changes in this branch and push them up, the PR will be updated. Once we're happy with the state of the PR, the flow is usually something like the reviewer hits the "approve" button and you hit the "merge" button.